lib/platform/pg/exporter.go

```diff
@@ -374,7 +402,7 @@ func (db DatabaseExporter) restoreCustomFile(exitOnError bool) error {
 			command.Stderr(stderrBuff),
 			command.Timeout(db.Timeout))...)
 	if err != nil {
-		return errors.Wrapf(err, "failed to import SQL file from %q, stderr: %s", source, stderrBuff.String())
+		return errors.Wrapf(err, "failed to import FC format file from %q, stderr: %s", source, stderrBuff.String())
```

Comment:
This is unrelated to the fix, but fixes an incorrect log statement (apparent copy pasta error)